### PR TITLE
libcap: disable GOLANG

### PR DESCRIPTION
**Pull request by a-schaefers**
Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## New package

- [ ] Latest upstream version.
- [ ] I agree to maintain this package.
- [ ] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/libcap/build [hunk 1]
```diff
@@ -6,6 +6,7 @@ sed -i 's/pthread_yield/sched_yield/'                    libcap/psx.c
 
 make
 make \
+    GOLANG=no \
     PREFIX=/usr \
     SBINDIR=/usr/bin \
```
Comment (dylanaraps) [inline community/libcap/build]: Does this work when run from `make install`?

Comment (a-schaefers) [reply community/libcap/build]: wait don't merge this, you caught something here

Comment (a-schaefers) [reply community/libcap/build]: should be good now. tested it builds with and without 'go' installed on the system.

File: community/libcap/build [hunk 2]
```diff
@@ -4,11 +4,16 @@ sed -i "s/CFLAGS :=/CFLAGS := \$(CFLAGS)/;/LDFLAGS :=/d" Make.Rules
...
-    DESTDIR="$1" \
-    RAISE_SETFCAP=no \
-    install
+mk() {
+    make \
+        GOLANG=no \
+        PREFIX=/usr \
+        SBINDIR=/usr/bin \
+        LIBDIR=/usr/lib \
+        DESTDIR="$1" \
```
Comment (dylanaraps) [inline community/libcap/build]: DESTDIR is duplicated here and below.

Comment (a-schaefers) [reply community/libcap/build]: Ah, fixed.

**Pull request merged**